packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -75,7 +75,7 @@ function buildBoundOperation(
   };
 }
 
-function splitMissingParameter(
+function splitBoundOperationsByMissingParameters(
```

Comment:
[req] As we are not looking at unbound operations anymore, the `if (!curr.IsBound)` condition can be removed.